.github/workflows/tests.yml

```diff
@@ -59,3 +59,33 @@ jobs:
                   path: |
                       coverage
                       !coverage/lcov-report
+
```

Comment:
Please could you update the name of this job to "Tests and SonarQube" or similar?

Thank you!